src/transformers/models/deta/modeling_deta.py

```diff
@@ -1416,12 +1416,12 @@ def get_decoder(self):
 
     # Copied from transformers.models.deformable_detr.modeling_deformable_detr.DeformableDetrModel.freeze_backbone
```

Comment:
```suggestion
```
the failing tests are due to these lines which no longer work! 

We should also make sure there is a test for this in the test modeling common, but this can be a follow up PR! 🤗

Reply:
@ArthurZucker I will accept your suggestion, but before that I can try make test modeling common.

Do you mean by adding some function in https://github.com/huggingface/transformers/blob/acd653164b6874e395ac9d46850f67599d8cdb58/tests/test_modeling_common.py#L156
or 
https://github.com/huggingface/transformers/blob/acd653164b6874e395ac9d46850f67599d8cdb58/tests/models/deta/test_modeling_deta.py#L184

such as
```
def test_freeze_backbone():

def test_unfreeze_backbone():
```

Reply:
Okay, let me build on this deta example and if it looks good then we can extend it to other table transformers.